### DISCUSSION TITLE
WatsonTCP on Windows 11, not connecting

### DISCUSSION THREAD

**Discussion by Mudoch** [jchristn/WatsonTcp#305] [General]
I'm having an issue with Server side not responding to Connection requests under windows 11, however the same code responds under windows 10.  Checked firewall and everything looks good. Tried running the app as admin under windows 11 too same result.  

I getting the message timeout trying to connect to ip address did not respond in time.

Again same code (exe) is running fine under windows 10. 

Any ideas on what I'm missing?  

.net 6.0 app, watsontcp version 4.8.14.14.  Tried to update to latest version but there I get breaking changes with WatsonTCP

    private static WatsonTcpServer _Server;

      _Server = new(null, _serverPort);
      _Server.Events.ClientConnected += ClientConnected;
      _Server.Events.ClientDisconnected += ClientDisconnected;
      _Server.Events.MessageReceived += MsgReceived;
      _Server.Callbacks.SyncRequestReceived += SyncRequestReceived;

      _Server.Settings.DebugMessages = false;
      _Server.Start();

**Comment by ShayanFiroozi** [comment]
Try disabling Windows Firewall ( also advanced firewall ) and run your app as administrator

**Reply by Mudoch** [reply]
Thank you for the reply.   I resolved this a while back and had forgotten I left this open.

The fix was rather simple.....  It turned out that the reserved IP was not resolving on the network. A reset of the router and removing then re-adding the MAC for reserved IP was what was needed.  I think my ISP had updated the router and caused a hang up in the reboot reading the saved configuration.
